website/docs/reference/scala-command/runner-specification.md

```diff
@@ -182,15 +182,15 @@ Set the Scala Native version (0.5.6 by default).
 
 **--native-mode**
 
-Set Scala Native compilation mode
```

Comment:
Everything under `website/docs/reference` is generated automatically with
```bash
./mill -i 'generate-reference-doc[]'.run
```

This needs to be adjusted in the Scala Native options implementation first:
https://github.com/VirtusLab/scala-cli/blob/4ab322768f7b5813312aeb553fed8a2caf35e4a4/modules/cli/src/main/scala/scala/cli/commands/shared/ScalaNativeOptions.scala#L22

Reply:
applied in 29f790e0546f2ee98ff4da92741672de6f322c5a.